kvdb-rocksdb/src/lib.rs

```diff
@@ -166,16 +166,17 @@ pub struct DatabaseConfig {
 	/// It can have a negative performance impact up to 10% according to
 	/// https://github.com/facebook/rocksdb/wiki/Statistics.
 	pub enable_statistics: bool,
-	/// Open the database as a secondary instance
-	/// secondary instances are read-only but kept up-to-date by tailing the rocksdb MANIFEST
+	/// Open the database as a secondary instance.
+	/// Specify a path for the secondary instance of the database.
+	/// Secondary instances are read-only and kept updated by tailing the rocksdb MANIFEST.
 	/// It is up to the user to call `catch_up_with_primary()` manually to update the secondary db.
-	/// disabled by default
+	/// Disabled by default.
 	///
-	/// must be opened with `max_open_files = -1`
-	/// may have negative performance on the secondary instance if the secondary instance applies log files
-	/// right before the primary instance performs a compaction
-	/// more info: https://github.com/facebook/rocksdb/wiki/Secondary-instance
-	pub secondary: bool,
+	/// Must be opened with `max_open_files = -1`.
+	/// May have a negative performance impact on the secondary instance
+	/// if the secondary instance applies the logs before the primary instance performs a compaction.
```

Comment:
This might be me being thick, but I don't understand what "applies the logs" mean, can you clarify?

Reply:
no problem! The logs is a bit of a vague term. I think replacing it with `if the secondary instance applies state changes before the primary instance compacts them` conveys the meaning better.

What I'm trying to get at is if the secondary instance reads the manifest logs before they have been compacted by rocksdb, then it is going to take longer to read through all of them and apply each of the changes to the secondary instance.